htdocs/api/v0.0.1/Login.php

```diff
@@ -75,9 +75,13 @@ function handlePOST()
         $login = $this->getLoginAuthenticator();
 
         if ($login->passwordAuthenticate($user, $password, false)) {
-            $this->JSON = array(
-                           "token" => $this->getEncodedToken($user),
-                          );
+            $token = $this->getEncodedToken($user);
+            if ($token) {
```

Comment:
I'd avoid truthiness comparisons in PHP or constructs where an average developer would need to look up the language spec to be sure of what's happening. `if (empty($token))` is probably sufficient here, since it's not somewhere you care about the usual problem of unexpectedly returning true for the string '0'..